src/pages/docs/octopus-rest-api/openid-connect/index.md

```diff
@@ -148,7 +148,35 @@ A Service Account Id will be shown, this will be a GUID which must be supplied a
 
 The access token obtained from the token exchange must be supplied in the `Authorization` header of API requests, using the `Bearer` scheme, for example `Authorization: Bearer {the-access-token}`.
 
-## Validation of OIDC tokens
+## Validation of OIDC identity tokens
+
+When an OIDC identity token from an external system is received as part of a token exchange request, Octopus will validate this token before issuing an access token. It does this by:
+
+- Matching the details of the token to an OIDC identity on an Octopus [service account](/docs/security/users-and-teams/service-accounts) using the audience (`aud`), issuer (`iss`) and subject (`sub`).
```

Comment:
These bullet points are grammatically inconsistent. They should either all be -ing verbs, or none.
e.g. Matching, Obtaining, Verifying
- OR - 
Matches, Obtains, Verifies

Reply:
Ah right good pickup, I'll make them consistent, ta.

Reply:
```suggestion
- Matches the details of the token to an OIDC identity on an Octopus [service account](/docs/security/users-and-teams/service-accounts) using the audience (`aud`), issuer (`iss`) and subject (`sub`).
```